app/migrations/Version20180116150221.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+declare(strict_types=1);
```

Comment:
It looks like we don't want to keep this on migration.

Reply:
ok